React/CoreModules/RCTAlertController.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <React/RCTUIKit.h> // TODO(macOS ISS#2323203)
+
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
```

Comment:
So something like I did initially here: https://github.com/microsoft/react-native-macos/pull/661#discussion_r535182016 but name it e.g. RCTPlatformAlertController? Should I also put it into RCTUIKit.h?

Reply:
Seeing as this header might taint other compilation units, it might be more appropriate to not define this alias for any referrer, but instead:

```objc
#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
@interface RCTAlertController : NSViewController
#else
@interface RCTAlertController : UIAlertController
#endif // ]TODO(macOS ISS#2323203)
```

After that this change is good to go from my end. @harrieshin has a good point, but that’s a discussion/change that should be made upstream first.

Reply:
Ok, addressed